tests/testthat/test-utils.R

```diff
@@ -60,5 +17,49 @@ testthat::test_that("format expression concatenates results of remove_enclosing_
       z <- 1
     })
   )
-  expect_equal(format_expression(code_list), c("x <- 1", "y <- 1", "z <- 1"))
+  expect_equal(format_expression(code_list), c("x <- 1", "y <- 1\nz <- 1"))
+})
+
+
+# lang2calls ------------------------------------------------------------------------------------------------------
+
+testthat::test_that("format_expression returns expression/calls into characters without curly brackets", {
+  expr1 <- expression({
+    i <- iris
+    m <- mtcars
+  })
+  expr2 <- expression(
+    i <- iris,
+    m <- mtcars
+  )
+  expr3 <- list(
+    expression(i <- iris),
+    expression(m <- mtcars)
+  )
+  cll1 <- substitute({
+    i <- iris
+    m <- mtcars
+  })
+  cll2 <- list(
+    quote(i <- iris),
+    quote(m <- mtcars)
+  )
+
+  # function definition
+  fundef <- quote(
+    format_expression <- function(x) {
+      x + x
+      return(x)
+    }
+  )
+
+  testthat::expect_identical(format_expression(expr1), "i <- iris\nm <- mtcars")
+  testthat::expect_identical(format_expression(expr2), c("i <- iris", "m <- mtcars"))
+  testthat::expect_identical(format_expression(expr3), c("expression(i <- iris)", "expression(m <- mtcars)"))
+  testthat::expect_identical(format_expression(cll1), c("i <- iris", "m <- mtcars")) # FAILS
+  testthat::expect_identical(format_expression(cll2), c("i <- iris", "m <- mtcars"))
+  testthat::expect_identical(
```

Comment:
I thought they would all be `"i <- iris\nm <- mtcars"` :thinking: 
Will have a look tomorrow.